com.playeveryware.eos/Runtime/Core/EOSManager.cs

```diff
@@ -378,19 +373,19 @@ public T GetOrCreateManager<T>() where T : IEOSSubManager, new()
                     manager = new T();
                     s_subManagers.Add(type, manager);
 
-                    if (manager is IEOSOnConnectLogin)
+                    if (manager is IEOSOnConnectLogin connectLogin)
```

Comment:
Is this syntax a new feature?

Reply:
Not terribly new - been there for several years if I'm not mistaken.